src/app/pages/category/category.component.scss

```diff
@@ -62,14 +56,14 @@
     .category-article {
       width: 95%;
       margin-bottom: $main-border;
-      border: $fairy-color 1px solid;
-      box-shadow: 10px 10px 10px $fairy-color;
-      transition: 0.3s ease-in-out;
-
-      &:hover {
-        border: $secondary-text-color 1px solid;
-        box-shadow: 10px 10px 10px $secondary-text-color;
-      }
+      // border: $fairy-color 1px solid;
```

Comment:
remove dead code

Reply:
This code is to keep for a later implementation. We will comment the code to express that need

Reply:
It's because of a problem with the article card itself which was highlighted by that so we removed it temporarily to give us time to find a way to fix it

Reply:
My bad, the comment wasn't displayed on the changes